nuxt/components/form/Form.vue

```diff
@@ -61,6 +68,26 @@ const props = withDefaults(defineProps<Props>(), {
   submitName: undefined,
 })
 
+const store = useMaevsiStore()
+const turnstileKey = ref('')
+const disabledSubmitButton = ref(true)
+
+watch(turnstileKey, (newKey) => {
+  if (newKey !== undefined || newKey !== '') {
+    disabledSubmitButton.value = false
+  }
+})
+
+function setTurnstileKeyAndEmit() {
+  setTurnstileKeyToStore()
+  emit('click')
+}
+
+function setTurnstileKeyToStore() {
+  consola.info('Got turnstile key: ' + turnstileKey.value)
+  store.turnstileKey = turnstileKey.value
+}
+
```

Comment:
Allows for short code similar to this:

```suggestion
const setTurnstileKeyToStore = () =>
    store.turnstileKey = turnstileKey.value

```

Reply:
I've applied the suggestion (54624035e325524baf55f360ffbdc1fbcec010d0)